src/libstd/net/addr.rs

```diff
@@ -1000,6 +1000,14 @@ impl ToSocketAddrs for (&str, u16) {
     }
 }
 
+#[unstable(feature = "string_u16_to_socket_addrs", issue = "73006")]
```

Comment:
Trait implementations are always stable.

Reply:
Oh I didn't realize! — what should this be marked with instead? `#[stable()]` with the next major?

Reply:
Yep - 1.46.0 specifically.

Reply:
Does these kind of change even needs a FCP? It looks minor but worthy enough to have a release note for it.